### ISSUE TITLE
The content of the tab page is black/frozen

### ISSUE THREAD

**Issue by KLGJ** [mozilla-mobile/fenix#21847]
> I opened two tabs:
> https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps
> https://mdn.github.io/pwa-examples/js13kpwa/
> The screen was turned off for a while, and when it was turned on again, the content of the first tab displayed abnormally: black, white, or the content of the second tab. 

我打开了两个标签页：
https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps
https://mdn.github.io/pwa-examples/js13kpwa/
屏幕关闭了一会儿，再次打开后，第一个标签页的内容显示异常：黑色、白色或者第二个标签页的内容。
![Screenshot_20211011_223346](https://user-images.githubusercontent.com/21209621/136809115-f003dd67-f676-408c-8262-22b0eecfdec0.jpg)
![Screenshot_20211011_223348](https://user-images.githubusercontent.com/21209621/136809123-954fe251-93b5-4fc5-be3a-18d127d91d28.jpg)
![Screenshot_20211011_223250](https://user-images.githubusercontent.com/21209621/136809129-9774fec9-9e89-498a-8097-3ce25518c4b8.jpg)
![Screenshot_20211011_223338](https://user-images.githubusercontent.com/21209621/136809134-2b753c99-d133-4dfc-a8e5-6cf535aaaa1c.jpg)
![Screenshot_20211011_223341](https://user-images.githubusercontent.com/21209621/136809144-6d16e070-9c40-41b1-aeaf-99f43455a59c.jpg)
![Screenshot_20211011_223344](https://user-images.githubusercontent.com/21209621/136809152-a686c03e-3d5f-4e9a-9712-31cb672720cf.jpg)

设备型号：vivo X23
Android 版本：Android 9
Firefox Nightly 版本：95.0a1 (Build #2015838891)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-18260)

**Comment by RuthlessRuler** [comment]
I am constantly having this issue. I generally use Private Mode, and I face the same behaviour.

Sometimes a tab just crash or gets corrupted. IDK the right word.
So If I switch tabs from Tab A to B using swipe gesture on address bar, I see the content of Tab A in tab B. But If I open another application, it will display blank


Now, If I open a new tab C, with some new content, and then using tab switcher to go back to tab B, I will see the content of Tab C in tab B.

So basically this corrupter Tab B is showing whatever content was there in the Previous tab used.

Refreshing the Tab B doesn't work nor does changing the address to another website works, as no action seems to be performed. Only way to fix this is to close the corrupted Tab.

**Comment by RuthlessRuler** [comment]
Hey, IDK why this has been marked closed. 
This still occurs from time to time
